server/api.go

```diff
@@ -32,23 +33,174 @@ type StartMeetingFromAction struct {
 	Context struct {
 		MeetingID    string `json:"meeting_id"`
 		MeetingTopic string `json:"meeting_topic"`
-		Personal     bool   `json:"personal"`
 		RootID       string `json:"root_id"`
+		Personal     bool   `json:"personal"`
 	} `json:"context"`
 }
 
+type JaaSSettingsFromAction struct {
+	Jwt  string `json:"jaasJwt"`
+	Path string `json:"jaasPath"`
+}
+
+func (p *Plugin) InitAPI() *mux.Router {
+	r := mux.NewRouter()
+
+	apiRouter := r.PathPrefix("/api/v1").Subrouter()
+
+	apiRouter.HandleFunc("/meetings/enrich", p.handleEnrichMeetingJwt).Methods(http.MethodPost)
+	apiRouter.HandleFunc("/meetings", p.handleStartMeeting).Methods(http.MethodPost)
+	apiRouter.HandleFunc("/config", p.handleConfig).Methods(http.MethodPost)
+	apiRouter.HandleFunc("/meetings/jaas/settings", p.handleJaaSSettings)
+	apiRouter.HandleFunc("/meetings/{apiid}/{roomname}", p.handleJaaSWindow)
+	r.HandleFunc("/jitsi_meet_external_api.js", p.handleExternalAPIjs)
+	r.HandleFunc("/jaas-main.js", p.handleJaaSBundle)
+	r.HandleFunc("{anything:*}", http.NotFound)
+
+	return r
+}
+
 func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
-	switch path := r.URL.Path; path {
-	case "/api/v1/meetings/enrich":
-		p.handleEnrichMeetingJwt(w, r)
-	case "/api/v1/meetings":
-		p.handleStartMeeting(w, r)
-	case "/api/v1/config":
-		p.handleConfig(w, r)
-	case "/jitsi_meet_external_api.js":
-		p.handleExternalAPIjs(w, r)
-	default:
+	p.router.ServeHTTP(w, r)
+}
+
+func (p *Plugin) handleJaaSWindow(w http.ResponseWriter, r *http.Request) {
+	if p.getConfiguration().UseJaaS {
+		if p.isJaaSMeeting(r.URL.Path) {
+			p.handleOpenJaaSMeeting(w, r)
+			return
+		}
+	}
+}
+
+func (p *Plugin) handleJaaSSettings(w http.ResponseWriter, r *http.Request) {
+	if !p.getConfiguration().UseJaaS {
+		mlog.Error("error JaaS requested while disabled")
```

Comment:
Check this everywhere log errors will be captial 
```suggestion
		mlog.Error("Error JaaS requested while disabled")
```